### DISCUSSION TITLE
How to make scroll of table?

### DISCUSSION THREAD

**Discussion by ghost** [CaiJimmy/hugo-theme-stack#805] [Q&A]
Hello. I want to make horizontal scroll of table. 
So I put "white-space: nowarp;" in "table". 

    table {
        white-space: nowrap;
        border-collapse: collapse;
        border-spacing: 0;
        margin-bottom: 1.5em;
        font-size: 0.9em;
    }

But after I put this code, I got horizontal empty space of mobile page and the scroll bar of window page was activated. 
How can I make scroll of table properly?

**Comment by zhixuan2333** [comment]
Slack is supported horizontal scroll Table.
https://dev.stack.jimmycai.com/p/markdown-syntax-guide/#inline-markdown-within-tables

**Reply by ghost** [reply]
Thank you for reply. 
But I don't want any words to be auto carriage return. I want to make a table with no carriage return unless I typed "<br />" in it. 

For example, if I write "Lorem ipsum dolor sit amet" inside of table and some other contenst too, it makes carriage return like this. 

Lorem 
ipsum 
dolor
sit
amet

I don't want this sitation. I want one long sentence like below. 

Lorem ipsum dolor sit amet

How can I solve this problem?

**Reply by zhixuan2333** [reply]
> But after I put this code, I got horizontal empty space of mobile page and the scroll bar of window page was activated.
How can I make scroll of table properly?

Can u tell me how to make the situation? Because I use the same code I got the normal nonwarp and scrollable table.
![image](https://user-images.githubusercontent.com/59254886/227872952-d55f56cf-8636-45af-9b04-a1e2b7ab9d51.png)

**Reply by ghost** [reply]
I tested and I figured out the condition that this happened. 
If you put LaTex at the end of a long text inside of the table, then some gap is created horizontally in total window. 
Just like this text...

| Table Title |
|---|
| Lorem ipsum dolor sit amet Lorem ipsum dolor sit amet Lorem ipsum dolor sit amet $x + x$ |
